notebooks/06_k_mean_clustering_on_sphere.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>Could add ; at the end of avoid printing the log.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>